scripts/go_install.sh

```diff
@@ -40,6 +40,6 @@ fi
 rm "${GOBIN}/${2}"* || true
 
 # install the golang module specified as the first argument
-go install -tags tools "${1}@${3}"
+go install -tags capztools "${1}@${3}"
```

Comment:
is this correct?

Reply:
I've reverted that change. I think it was an issue I had on macOS during debugging.